htdocs/install/mysql/migration/19.0.0-20.0.0.sql

```diff
@@ -404,3 +404,7 @@ ALTER TABLE llx_hrm_evaluation ADD COLUMN entity INTEGER DEFAULT 1 NOT NULL;
 ALTER TABLE llx_menu MODIFY COLUMN url TEXT NOT NULL;
 
 UPDATE llx_c_units SET short_label = 'mn' WHERE short_label = 'i' AND code = 'MI';
+
+ALTER TABLE llx_bank_categ RENAME TO llx_category_bank;
```

Comment:
Because this change is for develop branch and v21, this change must be added into migration/20.0.0-21.0.0.sql

Reply:
ok